src/utils/dispute-utils.js

```diff
@@ -53,6 +54,13 @@ export const transformDisputeDataAttributes = dispute => {
     }),
   }
 
+  // If the dispute is part of the precedence campaign we will flag it as such
+  const precedenceCamapignDisputes = getPrecedenceCampaignDisputesByCourt()
+  const isPartOfPrecedenceCampaign = precedenceCamapignDisputes.some(
+    disputeId => disputeId === dispute.id
```

Comment:
Not a big deal, but we might as well use a Map here like the voided ones. If not for performance, then just for consistency :).